Core/Graphics/Source/Android/GraphicsPlatform.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <bgfx/bgfx.h>
+
+// MSAA is disabled on Android.
+// See issue https://github.com/BabylonJS/BabylonNative/issues/494#issuecomment-731135918
+// for explanation
+#define BGFX_RESET_FLAGS (BGFX_RESET_VSYNC | BGFX_RESET_MAXANISOTROPY)
```

Comment:
Not for this PR, but we probably shouldn't use #define for this. It should be a constexpr like the code below.